apps/emacs/emacs.talon

```diff
@@ -204,8 +206,9 @@ search toggle case [fold | sensitive]: user.emacs_meta("c")
 search toggle regex: user.emacs_meta("r")
 
 highlight lines matching [regex]: user.emacs("highlight-lines-matching-regexp")
+highlight phrase: user.emacs("highlight-phrase")
 highlight regex: user.emacs("highlight-regexp")
-unhighlight regex: user.emacs("unhighlight-regexp")
+unhighlight (regex | phrase): user.emacs("unhighlight-regexp")
```

Comment:
Just noticed we have regex here and regex**p** elsewhere. Do you pronounce the "p"? I find it gets eaten a bit if I try to say it out loud. Elsewhere in community we use "expression" (e.g. hunt expression: user.find_toggle_match_by_regex()). I'm OK with this particular change though, makes sense! Perhaps consider `(regex | expression)`.

Reply:
I don't think I pronounce the p, but I don't think I've ever noticed a misrecognition either way. Would make sense to standardize on `regex` in the spoken forms, I suppose (emacs itself consistently uses `regexp`).

Re `expression`, I'd tend to prefer to stick close to the naming of the emacs commands generally. But if folks want to consistently add `expression` as an alternative everywhere, that'd be fine. Probably its own PR though.

Reply:
I'm OK with just changing everything to `regex`; if other people want `expression` they can file another PR/ticket!

Reply:
done